openedx/adg/admin_dashboard/admin_settings/tests/test_views.py

```diff
@@ -0,0 +1,57 @@
+"""
+Tests for `admin_settings` app views
+"""
+import pytest
+
+from common.djangoapps.student.tests.factories import GroupFactory, UserFactory
+from openedx.adg.admin_dashboard.admin_settings.models import Invitation
+from openedx.adg.admin_dashboard.admin_settings.tests.factories import InvitationFactory
+from openedx.adg.admin_dashboard.admin_settings.views import AdminAccountListView, PendingInvitationListView
+from openedx.adg.lms.applications.constants import ADG_ADMIN_GROUP_NAME
+from openedx.adg.lms.applications.tests.factories import BusinessLineFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+
+@pytest.mark.parametrize('total_admins, expected_count', [
+    (5, 10),
+    (10, 20),
+])
+@pytest.mark.django_db
+@skip_unless_lms
+def test_admin_accounts_list_queryset(total_admins, expected_count):
+    """
+    Tests queryset for `AdminAccountListView` returns expected number of active admins.
+    """
+    business_line = BusinessLineFactory()
+    adg_admin_group = GroupFactory(name=ADG_ADMIN_GROUP_NAME)
+
+    for _ in range(total_admins):
+        # pylint: disable=no-member
+        user = UserFactory()
+        user.groups.add(adg_admin_group)
+
+        user = UserFactory(is_active=True)
+        user.groups.add(business_line.group)
+
+        user = UserFactory(is_active=False)
+        user.groups.add(business_line.group)
+        # pylint: enable=no-member
```

Comment:
Maybe we can use `create_batch` instead of loop.

Reply:
Will have to use 3 different loops again to add groups so I think this one is better. Going to use `create_batch` for the other test.